modules/navier_stokes/test/tests/finite_volume/ins/natural_convection/diff_heated_cavity_linear_segregated.i

```diff
@@ -0,0 +1,250 @@
+################################################################################
```

Comment:
I think a better place for this would be `test/test/finite_volume/ins/natural_convection/linear_segregated/*` We might have more cases coming here so I think adding one more layer of folders could be nice.

Reply:
done